sql/wsrep_sst.cc

```diff
@@ -1580,6 +1614,37 @@ int wsrep_sst_donate(const std::string &msg, const wsrep::gtid &current_gtid,
     return WSREP_CB_FAILURE;
   }
 
+
+  /*
+  [start]
+  section to support clone user/pw
+  check for auth@addr separator
+  */
+  const char* addr= strrchr(data, '@');
+  wsp::string remote_auth;
+  if (addr)
+  {
```

Comment:
code formating

Reply:
@kamil-holubicki for the code formatting I am trying to follow the original code (Codership) if you check the wsrep_sst.cc and othr you will see them following the code style 
if
{}
and  to have one param per line in the function declaration. 
for me is the same but I thought more correct to follow the originator way. 
what you think?

Reply:
Hard to say which code is the original one :). Please use clang formatted as we discussed.

Reply:
done